spec/models/manageiq/providers/kubernetes/container_manager/container_group_spec.rb

```diff
@@ -58,5 +65,8 @@ def assert_volumes_relations(group, ems, container_volume, persistent_volume)
     expect(persistent_volume.parent.class).to eq(ManageIQ::Providers::Kubernetes::ContainerManager)
     expect(persistent_volume.parent.name).to eq("ems")
     expect(persistent_volume.parent_type).to eq("ExtManagementSystem")
+    unless persistent_volume.container_groups.empty?
```

Comment:
@nimrodshn I prefer not to use this condition in this test. Can you move this check so we don't need it?